docs/bluetooth/MicroBitPartialFlashing.md

```diff
@@ -0,0 +1,147 @@
+# Partial Flashing Service
+
+## Introduction
+
+The Partial Flashing Service allows a BLE client to connect to a micro:bit and read and write the information required to partially update the firmware. For MakeCode programs this is the user's script, and for Python programs this is the file system.
+
+This service exists in partnership with the DFU service that allows a client to fully flash the micro:bit.
+
+Diagrams showing the flow of data during partial flashing can be found at [the bottom of the document](#partial flashing client).
+
+## Bluetooth Service Specification
+
+TODO: Does BLE profile exist for v2?
```

Comment:
For V2 we're confirming to the same profile spec as we did for V1, so linking to same place is fine. We haven't got an updated version.
Do we need to modify that spec to update it for the small variation between V1 and V2?

Reply:
The DFU service has changed to the up to date Nordic DFU